third_party/docfx/templates/devsite/partials/uref/namespace.tmpl.partial

```diff
@@ -216,14 +216,9 @@
         {{#children}}
           <h4 id="{{id}}" data-uid="{{uid}}">{{name.0.value}}</h4>
           <div class="markdown level1 summary">{{{summary}}}</div>
-          <div class="markdown level1 conceptual">{{{conceptual}}}</div>
           {{#syntax}}
           <h5 class="decalaration">{{__global.declaration}}</h5>
-          <div class="codewrapper">
-            <pre><code class="prettyprint">{{syntax.content.0.value}}</code></pre>
-          </div>
           {{/syntax}}
-          {{>partials/uref/namespace.examples}}  

```

Comment:
This one seems the most suspicious to me... But, do all of these removed lines cause the issue?

Reply:
Yes, not just the `partials`.